### PR TITLE
Dev4

### PR DESCRIPTION
**Pull request by Newton-Ephraim**
sorry i didn't deliver all these while, abeg no vex.

### CODE REVIEW

**Comment by Newton-Ephraim** [comment]
check for any correction please

File: src/assets/images/Rectangle 90.png
Comment (KoneeVibes) [inline src/assets/images/Rectangle 90.png:1]: You should name this file rightly. The whitespace between "Rectangle" and "90" is causing the entire application to break on my end.

**Review by KoneeVibes — CHANGES_REQUESTED** [review]
This get started page is not responsive. It is also not a replica of the design. Please take some time to clean this up.

Here are some concerns:

image
image

**Review by KoneeVibes — CHANGES_REQUESTED** [review]
Also, you have not completed this auth page. You see, considering that you have now used 4 weeks on this task, I would have expected that you just cleaned this up since Awele is not available.

**Review by KoneeVibes — APPROVED** [review]

**Pull request merged**